doc/Known_issues/Troubleshooting.md

```diff
@@ -11,12 +17,34 @@ for script configuration info.
 **Note**: the script can be ran with or without Administrator privileges. However, some checks
```

Comment:
Why is this note here not in the diagnostic check readme?

Reply:
Good point, I'll add it there.